app/services/caregivers_assistance_claims_service.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class CaregiversAssistanceClaimsService
+  def submit_claim!(user_context, claim_data)
+    claim = SavedClaim::CaregiversAssistanceClaim.new(claim_data)
+
+    unless claim.valid?
+      # TODO: (kevinmirc #5672) - increment failure.client stat
```

Comment:
We strongly prefer leaving TODOs out of the codebase.

Maybe you could link to the file/code location from the issue instead of the other way around?